packages/browser-wallet/src/popup/store/identity.ts

```diff
@@ -65,26 +65,28 @@ export const setRecoveryPayloadAtom = atom<null, RecoveryPayload, Promise<void>>
     set(recoveryStatusAtom, { payload })
 );
 
-export const identityByAddressAtomFamily = atomFamily<string, Atom<AsyncWrapper<ConfirmedIdentity | undefined>>>(
-    (address) =>
-        atom((get) => {
-            const creds = get(credentialsAtomWithLoading);
-            const ids = get(identitiesAtomWithLoading);
+export const identityByAddressAtomFamily = atomFamily<
+    string | undefined,
+    Atom<AsyncWrapper<ConfirmedIdentity | undefined>>
+>((address) =>
+    atom((get) => {
+        const creds = get(credentialsAtomWithLoading);
+        const ids = get(identitiesAtomWithLoading);
 
-            if (creds.loading || ids.loading) {
-                return { loading: true, value: undefined };
-            }
+        if (creds.loading || ids.loading) {
+            return { loading: true, value: undefined };
+        }
```

Comment:
Is it just formatting of the first line that caused the change in indent? (Can't find another reason)

Reply:
it's the change in the type from `string` to `string | undefined` in the first type param of `atomFamily`